examples/spmm/spmm.cc

```diff
@@ -532,7 +532,7 @@ class SpMM {
       auto A = baseT::template get<0>(_ijk);
       auto B = baseT::template get<1>(_ijk);
 #if BLOCK_SPARSE_GEMM
-      Blk zero(btas::Range(A.range().extent(0), A.range().extent(1)), 0.0);
```

Comment:
I think it was correct with A.range().extent(0) and B.range().extent(1): the tile we want to initialize to 0 is the tile of C, which should be mb x nb. The input tiles are A (mb x kb) and B (kb x nb). So... yes, I want zero to be the first dimension of A in its first dimension, and the second dimension of B in its second dimension.